File: docs-site/python/close.py
def callback_fn(data):
    print(data)

async def on_connected():
  # ... Code from previous step

  unsubscribed = await client.off("chat.room1")
  print(f"Unsubscribed from chat.room1 => {unsubscribed}")

  await client.close()

await client.on("chat.room1", callback_fn)
await client.on(Realtime.CONNECTED, on_connected)

client.connect()

File: docs-site/nodejs/close.js
await client.on("chat.room1", (data) => {
    console.log(data);
})

client.on(Realtime.CONNECTED, async () => {
  // ... Code from previous step

  var unsubscribed = await client.off("chat.room1")
  console.log("Unsubscribed from chat.room1 => " + unsubscribed)

  await client.close()
})

client.connect()